hack/cluster-configs/config-marvell-dpu.yaml

```diff
@@ -0,0 +1,23 @@
+# This deploys the Marvell DPU itself, installs microshift and the dpu-operator.
+# Run on hosts:
+# - wsfd-advnetlab44.anl.eng.bos2.dc.redhat.com (Cluster 15: Marvell dev cluster)
+# - wsfd-advnetlab41.anl.eng.bos2.dc.redhat.com (Cluster 16: Marvell dev cluster)
+clusters:
+  - name : "iso_cluster"
+    api_vip: "192.168.122.99"
+    ingress_vip: "192.168.122.101"
+    network_api_port: "eno2"
+    kind: "iso"
+    install_iso: "rhel:9.4"
+    masters:
+    - name: "marvell-dpu-{{worker_number(0)}}"
+      node: "{{worker_name(0)}}"
+      kind: "marvell-dpu"
```

Comment:
This line is needed because the Marvell doesn't have a bmc?

Reply:
No. It's need because CDA needs to know which kind of iso-cluster to deploy.

Reply:
> No. It's needed because CDA needs to know which kind of iso-cluster to deploy.

it's needed, because the deployoment for a iso-cluster works differently with Marvell DPU vs. IPU. Part of the reason for that is that Marvell doesn't have a BMC (yet) and we use pxeboot to install the Marvell DPU.

Hence, CDA needs to know which kind of approach to take, and it determines it based on the node kind `marvell-dpu` vs. `physical`.